admin/class-tools.php

```
@@ -0,0 +1,448 @@
+        $encode_data = file_get_contents( $file );
+        $options     = json_decode( $encode_data, true );
+
+        $errors = new WP_Error();
+
+        if ( ! is_array( $options ) ) {
+            $errors->add( 'invalid_json', __( 'The uploaded file is not a valid WPUF export.', 'wp-user-frontend' ) );
+
+            return $errors;
+        }
+
+        foreach ( $options as $key => $value ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)